src/compat/db_json.cpp

```diff
@@ -1578,6 +1605,14 @@ db_json_split_path_by_delimiters (const std::string &path, const std::string &de
       tokens.push_back (substring);
     }
 
+  for (int i = 0; i < tokens.size(); i++)
```

Comment:
can you please make this unsigned int so that the compiler doesn't complain? or use std::vector<std::string>::size_type